source/tutorials/pnp-testsuites/pnp-testsuites.rst

```diff
@@ -0,0 +1,545 @@
+      Set a longer collection period (e.g. several minutes) to allow the system enters
+      deep sleep states.
+
+    * Active workload, such as video or audio playback. Launch the test programs and
+      start collecting data for a predetermined time period, or manually stop the data
+      collection. Use the :command:`help` command to learn various options to start/stop
+      the data collection.
+
+Our recommended testing method is:
+
+#. Push the Intel SoC Watch package to the DUT.
+
+    .. code-block:: bash
+
+        $ adb push socPowerTool /data/
+
+#. Run the ``socPowerTool`` command and specify a collection time period (e.g. 1800 seconds)
+   in the ``adb`` shell.
+
+    .. code-block:: console
+
+        $ adb shell
+        celadon:/ # /data/socPowerTool --get_power_once 1800
+
+#. The system returns the average power consumption during the specified time period.
+
+    .. figure:: images/socwatch-example.png
+        :align: center
+
+#. Learn more on the additional data that can be collected on DUT by entering
+   the :command:`help` command.
+
+    .. code-block:: console
+
+        celadon:/ # /data/socPowerTool -h
+
+Test Suite
+==========
+
+This section describes how to run the workloads included in the |C| Power and Performance (PnP)
+test suite and the Key Performance Indicators (KPIs) measured during the tests.
+
+H.264 3840x2160 60fps Playback
+------------------------------
+
+This KPI measures frame rate for video playback of a locally stored video clip using HSC.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Set up HSC environment.
+
+**Measurement Steps**
+
+    #. Download the H.264 3840x2160 60fps video source and push it into the device.
+
+    #. Play the video using the Gallery application or other video playback application.
+
+    #. Calculate FPS using HSC.
+
+Peak 1080p H.264 Decoding FPS
+-----------------------------
+
+This KPI measures the capability of parallel multiway decoding using an MSDK sample.
+For example, this could be built with the command: 
+
+.. code-block:: bash
+
+    $ make -j8 BOARD_HAVE_MEDIASDK_OPEN_SOURCE=true sample_decode
+
+The output is located in :file:`out/target/product/cel_kbl/vendor/bin` folder.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Compile or download the decoder sample file.
+
+    #. Download or create a H.264 testing video clip (FFmpeg can be used).
+
+**Measurement Steps**
+
+    #. Push the H.264 video clip and the compiled decoder sample binary file to the test device.
+
+    #. Set the decoder binary file executable, and execute the decoder sample binary file to get FPS.
+       Since the decoder program may not fully utilize 100% of the decoding engine's capability
+       on many platform, it's recommended to create a script to run multiple (e.g. 4) instances
```

Comment:
Change to:
Since the decoder program may not fully utilize 100% of the decoding engine's capability on many platforms, it's recommended to create a script to run multiple instances (e.g. 4)